### PR TITLE
docs: add instructions for uninstalling the Windows auth package

### PR DESCRIPTION
**Pull request by zmb3**

### CODE REVIEW

File: docs/pages/management/admin/uninstall-teleport.mdx [hunk 1]
```diff
@@ -187,6 +186,8 @@ Follow the instructions for your Linux distribution:
   ```
 (!docs/pages/includes/uninstall-teleport-connect-windows.mdx!)
 
+(!docs/pages/includes/uninstall-windows-auth.mdx!)
```
Comment (zmb3) [inline docs/pages/management/admin/uninstall-teleport.mdx:189]: @ptgott any idea why this partial is not showing up in the preview?

Comment (ptgott) [reply docs/pages/management/admin/uninstall-teleport.mdx:189]: It seems to work on my end!
image

**Review by ptgott — APPROVED** [review]

File: docs/pages/management/admin/uninstall-teleport.mdx (hunk 1 above)
Comment (zmb3) [reply docs/pages/management/admin/uninstall-teleport.mdx:189]: Odd. I'm certain I checked this page and it did not show up.

**Comment by zmb3** [comment]
@mmcallister would you mind reviewing this one?

**Review by mmcallister — APPROVED** [review]

**Comment by mmcallister** [comment]
Apologies for missing this the other day!

**Pull request merged**